### ISSUE TITLE
Docker Container Stops on Unraid

### ISSUE THREAD

**Issue by tacticalkeyboard** [Lissy93/web-check#144]
The docker container appears to stop on its own without interaction when installed on Docker/Unraid. The container needs to be manually started again. 

Edit:
Here's a copy of the logs before the container shut off.

node:events:491
      throw er; // Unhandled 'error' event
      ^
Error: connect ETIMEDOUT 8.8.4.4:443
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1278:16)
Emitted 'error' event on ClientRequest instance at:
    at TLSSocket.socketErrorListener (node:_http_client:494:9)
    at TLSSocket.emit (node:events:513:28)
    at emitErrorNT (node:internal/streams/destroy:157:8)
    at emitErrorCloseNT (node:internal/streams/destroy:122:3)
    at processTicksAndRejections (node:internal/process/task_queues:83:21) {
  errno: -110,
  code: 'ETIMEDOUT',
  syscall: 'connect',
  address: '8.8.4.4',
  port: 443
}
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

**Comment by MagicalSpacePope** [comment]
Also on Unraid, mine stops after each lookup. I'll start it and leave it unused to see if it stops on its own for me as well.

**Comment by MagicalSpacePope** [comment]
I have really struggled to get to the bottom of this one. Web-check dies every single time it's used on unraid, I cant figure out why.

**Comment by ther3zz** [comment]
you guys ever figure out why the container stops in unraid?

**Comment by MagicalSpacePope** [comment]
Nope! I just cant keep it running. I've debugged as much as I could, never figured it out. :(

**Comment by ther3zz** [comment]
> Nope! I just cant keep it running. I've debugged as much as I could, never figured it out. :(

add this to the extra params field in unraid: 
 --restart always

basically just restarts the container when it shuts off

**Comment by Jeremie-Anderlin** [comment]
Same issue in Container Manager on a Synology. Auto-restart as workaround, but still not ideal.
